src/scripts.js

```diff
@@ -117,7 +157,7 @@ function returnDatesOfWeek(userId, date) {
   let index = userData.findIndex((data) => data.date === date);
   return userData.splice(index - 6, 7).map(day => day.date);
 }
-
+function allGraphs() {
```

Comment:
Smart move wrapping the graphs in this function to ensure the fetch occurs first.